cmd/eventlistenersink/main.go

```diff
@@ -101,5 +102,15 @@ func main() {
 		w.WriteHeader(200)
 		fmt.Fprint(w, "ok")
 	})
-	logger.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", sinkArgs.Port), nil))
+
+	srv := &http.Server{
+		Addr:         fmt.Sprintf(":%s", sinkArgs.Port),
+		ReadTimeout:  sinkArgs.ELReadTimeOut * time.Second,
```

Comment:
ReadTimeout includes reading both headers + body. Maybe we should use `ReadHeaderTimeout` instead?

Reply:
Or may we set both? What would be the sane values for all these?

Reply:
5 seconds sounds reasonable to me.